middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -3869,29 +3910,19 @@ let wrap_final_module_block acc env ~program ~prog_return_cont
         (Bound_pattern.static bound_static)
         named ~body:return
     in
-    let block_access : P.Block_access_kind.t =
-      Values
-        { tag = Known Tag.Scannable.zero;
-          size =
-            Known
-              (Target_ocaml_int.of_int (Acc.machine_width acc)
-                 module_block_size_in_words);
-          field_kind = Any_value
-        }
-    in
     List.fold_left
       (fun (acc, body) (pos, var, var_duid) ->
         let var = VB.create var var_duid Name_mode.normal in
         let pat = Bound_pattern.singleton var in
-        let pos = Target_ocaml_int.of_int (Acc.machine_width acc) pos in
+        let pos' = Target_ocaml_int.of_int (Acc.machine_width acc) pos in
```

Comment:
Let's pick a better name here.  Maybe `field` since that's the name of the record field?